ui/prometheus-plugin/src/plugins/prometheus-time-series-query/PrometheusTimeSeriesQueryEditor.tsx

```diff
@@ -58,10 +58,11 @@ export function PrometheusTimeSeriesQueryEditor(props: PrometheusTimeSeriesQuery
       <TextField
         fullWidth
         label="Series Name Format"
```

Comment:
I changed this to "Name".

That might be controversial, but my thinking was:

- people don't know what we mean when we say "Series Name Format"
- `Name` is simple, and pretty close to what's happening (giving a name to each series)
- The example and helper text provide details to make it clear
- It covers the fact that we don't always use the formatting with `{{ }}`, sometimes we just enter text

Reply:
I'm good with this! It matches how it's defined in the JSON close enough and really like your suggestion for `Tip`